testing/activities/activity_definitions/main.tf

```diff
@@ -93,6 +93,32 @@ resource "leanspace_activity_definitions" "test" {
       type  = "TIME"
     }
   }
+  metadata {
+    name        = "ActivityMetadataBinary"
+    description = "A binary metadata value"
+    attributes {
+      value = "62696e617279"
+      type  = "BINARY"
+    }
+  }
+  metadata {
+    name        = "ActivityMetadataGeoPoint"
+    description = "A geopoint metadata value"
+    attributes {
+      type = "GEOPOINT"
+      fields {
+        elevation {
+          value = 141.0
+        }
+        latitude {
+          value = 48.5
+        }
+        longitude {
+          value = 7.7
+        }
+      }
+    }
+  }
```

Comment:
I have also tested that terraform init fails when specifying data types not supported by activity definitions (same for command definitions) - I don't know what else I can test; any idea is welcome!